error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/JUnitParameterizedMethodDeclaration.java

```diff
@@ -0,0 +1,48 @@
+package tech.picnic.errorprone.bugpatterns;
+
+import static com.google.errorprone.BugPattern.LinkType.CUSTOM;
+import static com.google.errorprone.BugPattern.SeverityLevel.SUGGESTION;
+import static com.google.errorprone.BugPattern.StandardTags.SIMPLIFICATION;
+import static com.google.errorprone.matchers.ChildMultiMatcher.MatchType.AT_LEAST_ONE;
+import static com.google.errorprone.matchers.Matchers.annotations;
+import static com.google.errorprone.matchers.Matchers.isType;
+import static tech.picnic.errorprone.bugpatterns.util.Documentation.BUG_PATTERNS_BASE_URL;
+
+import com.google.auto.service.AutoService;
+import com.google.errorprone.BugPattern;
+import com.google.errorprone.VisitorState;
+import com.google.errorprone.bugpatterns.BugChecker;
+import com.google.errorprone.bugpatterns.BugChecker.MethodTreeMatcher;
+import com.google.errorprone.matchers.Description;
+import com.google.errorprone.matchers.Matcher;
+import com.sun.source.tree.MethodTree;
+
+/**
+ * A {@link BugChecker} that flags test methods using {@link
+ * org.junit.jupiter.params.ParameterizedTest} without actually having any arguments.
+ */
+@AutoService(BugChecker.class)
+@BugPattern(
+    summary = "JUnit parameterized test used without arguments",
+    link = BUG_PATTERNS_BASE_URL + "JUnitParameterizedMethodDeclaration",
+    linkType = CUSTOM,
+    severity = SUGGESTION,
+    tags = SIMPLIFICATION)
+public final class JUnitParameterizedMethodDeclaration extends BugChecker
```

Comment:
Proposed name: `JUnitNullaryParameterizedTestDeclaration`. (Because we're not validating other `@ParameterizedTest` constraints (such as that they're at least one argument source) and because in practice the flagged methods aren't parameterized.)